apps/wallet/src/ui/app/staking/usePendingDelegation.tsx

```diff
@@ -9,8 +9,8 @@ import { notEmpty } from '../helpers';
 import { useAppSelector } from '../hooks';
 import { api } from '../redux/store/thunk-extras';
 
-const STATE_OBJECT = '0x5';
-const VALDIATOR_NAME = /^[A-Z-_.\s0-9]+$/i;
+export const STATE_OBJECT = '0x5';
+export const VALDIATOR_NAME = /^[A-Z-_.\s0-9]+$/i;
```

Comment:
Maybe just generalize some `getName` function here too to de-dupe the name logic?

Reply:
done